app/controllers/api/restful_controller.rb

```diff
@@ -3,6 +3,10 @@ class API::RestfulController < API::BaseController
   rescue_from CanCan::AccessDenied                    do |e| respond_with_standard_error e, 403 end
   rescue_from ActionController::UnpermittedParameters do |e| respond_with_standard_error e, 400 end
 
+  def show
```

Comment:
Finally moved this to the restful_controller; not sure why we didn't do that before. :)